klippy/toolhead.py

```diff
@@ -572,8 +572,8 @@ def cmd_SET_VELOCITY_LIMIT(self, gcmd):
                "max_accel: %.6f\n"
                "max_accel_to_decel: %.6f\n"
                "square_corner_velocity: %.6f"% (
-                   max_velocity, max_accel, self.requested_accel_to_decel,
-                   square_corner_velocity))
+                   self.max_velocity, self.max_accel, self.max_accel_to_decel,
+                   self.square_corner_velocity))
```

Comment:
I can make this a separate PR. The reason I added it to this PR initially was because:

1. Currently Klipper reports back the parameters from the request, even if it didn't apply them. Then the message is not particularly helpful. I have seen quite some confusion from the users (myself included at first), who believed that their limit was applied. This can be especially problematic if the user wanted to, but forgot to change the limit in their config (or did that but forgot to run `RESTART`) - they will believe they are doing the right thing without a good way to detect their mistake. So I think it would be great to fix that.

2. Resonance tester code issues a command to increase the maximum acceleration (and set accel-to-decel = max_accel); here arguably the user didn't request any limit change directly, so it is best to report the actually configured new limits. I was thinking that this can lead to some extra confusion if the user configured, say, `max_accel = 5000` because they know this is how much their printer can handle, and then they'll see that max_accel was set to 9000 (which would be 75 * 120), and can freak out with 'My printer will fall apart from this!' thought. That's why I put this change into this PR (but I'm happy to commit it separately).

Now why I made the change to `get_status` method: that method is used in logging, AFAICT. If it isn't updated, but this code is changed, this will lead to inconsistent messages in the logs. Please let me know how you think about updating this code and leaving `get_status` method be.

Reply:
I understand.  I think it would be best to do these changes separately from resonance tester - as resonance testing is new functionality and these changes alter existing functionality.  Since they're not directly tied to each other, I think it would be easier to commit one and then the other.  (And if there is a regression it will then be easier to rollback one without having to rollback both.)

Reply:
I moved these changes to a separate PR #3422.